include/osquery/events.h

```diff
@@ -20,7 +20,6 @@
 #include <boost/thread/locks.hpp>
 #include <boost/thread/mutex.hpp>
 
-#include <osquery/database.h>
```

Comment:
This was the only hashmap dependency I could find, which comes from RocksDB. Moving this from the public headers into the implementation removes the max/min overload from YARA until their next >3.3.0 release.